interfaces/HO-Portal/src/app/models/q-and-a.models.ts

```diff
@@ -0,0 +1,33 @@
+export enum AnswerType {
+  // Translate the types used in the API to internal, proper types:
+  Number = 'numeric',
+  Text = 'text',
+  Date = 'date',
+  Enum = 'dropdown',
+  phoneNumber = 'tel',
+  email = 'email',
+}
```

Comment:
You could also include this definition in the `fsp.model.ts`-file, because it is only related to that.

The whole "Q-and-A"-concept doesn't really exist in the HO-Portal.